Assignment4/jcdlg-a180369/task06.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+"""Task06.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/13LtDpsDvI9mkO52dxjGcT5DIn3LlEcSI
+
+**Task 06: Modifying RDF(s)**
+"""
+
+!pip install rdflib 
```

Comment:
This will not run in Python. You need to comment out line 12. The same applies for the other tasks